cross-wiki-weekend-2021.php

```diff
@@ -0,0 +1,296 @@
+<?php
+$title = 'Cross-Wiki Weekend 2021';
+include('_header.php');
+?>
+
+<style>
```

Comment:
Consider moving to stylesheet.

Reply:
Removed, and made links in the `.main` class now red by default across the site.